src/lib/y2storage/guided_proposal.rb

```diff
@@ -139,11 +139,11 @@ def calculate_proposal
             exception = error
           end
 
-          return true unless exception
+          return unless exception
```

Comment:
What was wrong with returning nil?

Reply:
Returning `true` implies that the return value matters. Then you have to check the code what's really expected. That's just annoying.

Reply:
I see - rubocop dislikes it. That's just insane.

Reply:
And does it really matter? I mean, with the current code it can only return `nil` (or `true` in your version) or raise an exception. So with the current method API, the returned value looks indeed irrelevant to me.

Don't get me wrong, I'm not against returning nil, true or `:I_did_it` :-) I simply don't see why the returned value is important with the current definition in which this is more a "procedure" than a "function", sort to say.

Reply:
Exactly. That's why I wanted to get **rid** of `true`.

Reply:
Oh, damn. I have been reading it the other way around all the time, like if you were ADDING the true. Sorry. Looks like I can't distinguish red from green (has happened to me twice this month).